doc/nrf/known_issues.rst

```diff
@@ -587,6 +587,15 @@ MCUboot
 Recovery with the USB does not work
   The MCUboot recovery feature using the USB interface does not work.
 
+USB
```

Comment:
Should this be a new section, or should it go in the Zephyr section?

Reply:
Zephyr section, will resubmit.